### DISCUSSION TITLE
UDM Pro - IPTV stops 2-3 minutes after install

### DISCUSSION THREAD

**Discussion by APellie** [fabianishere/udm-iptv#77] [Q&A]
Hi @fabianishere, 

Did the installation of v2.0.5 release on the default network (LAN) of the UDM Pro.
Everything works like a sunshine :-)

Step two was to install a seperate network (IPTV) on the UDM Pro.
Created the network with VLAN 44 (also tried 4 before) configured, IGMP Snooping on and DHCP options as mentioned in the install procedure. Its result: after few minutes IPTV stops.

**Setup information:**
ISP: KPN
Router: UniFi OS UDM Pro 1.11.3.3937
Switch: UniFi US-48-750W (newest firmware)

**Diagnostic information (not working situation):**
root@ubnt:/# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN Ranges: 213.75.0.0/16 217.166.0.0/16
WAN VLAN: 4 (dev iptv)
WAN DHCP options: -O staticroutes -V IPTV_RG
LAN Interfaces: br44
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
40: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue qlen 1000
    inet 10.194.4.208/20 brd 10.194.15.255 scope global iptv
       valid_lft forever preferred_lft forever
10.194.0.0/20 dev iptv  src 10.194.4.208
213.75.112.0/21 via 10.194.0.1 dev iptv  metric 240
=== Service Logs ===
Feb 23 12:11:23 ubnt udm-iptv[29897]: RECV Membership query   from 192.168.2.1     to 224.0.0.1
Feb 23 12:11:26 ubnt udm-iptv[29897]: RECV V2 member report   from 192.168.2.1     to 224.0.0.2
Feb 23 12:11:26 ubnt udm-iptv[29897]: The IGMP message was from myself. Ignoring.
Feb 23 12:11:28 ubnt udm-iptv[29897]: RECV V2 member report   from 192.168.2.1     to 233.89.188.1
Feb 23 12:11:28 ubnt udm-iptv[29897]: The IGMP message was from myself. Ignoring.
Feb 23 12:11:28 ubnt udm-iptv[29897]: The source address 192.168.2.1 for group 233.89.188.1 is from downstream VIF[0]. Ignoring.
Feb 23 12:11:29 ubnt udm-iptv[29897]: RECV V2 member report   from 192.168.2.1     to 224.0.0.22
Feb 23 12:11:29 ubnt udm-iptv[29897]: The IGMP message was from myself. Ignoring.
Feb 23 12:11:30 ubnt udm-iptv[29897]: RECV V2 member report   from 192.168.2.1     to 224.0.0.251
Feb 23 12:11:30 ubnt udm-iptv[29897]: The IGMP message was from myself. Ignoring.

**Diagnostic information (working situation):**
root@ubnt:/# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN Ranges: 213.75.0.0/16 217.166.0.0/16
WAN VLAN: 4 (dev iptv)
WAN DHCP options: -O staticroutes -V IPTV_RG
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
43: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue qlen 1000
    inet 10.194.0.154/20 brd 10.194.15.255 scope global iptv
       valid_lft forever preferred_lft forever
10.194.0.0/20 dev iptv  src 10.194.0.154
213.75.112.0/21 via 10.194.0.1 dev iptv  metric 243
=== Service Logs ===
**Feb 23 12:43:57 ubnt udm-iptv[19295]: Adding MFC: 213.75.167.58 -> 224.3.2.6, InpVIf: 1**
Feb 23 12:43:58 ubnt udm-iptv[19295]: The source address 192.168.1.1 for group 233.89.188.1 is from downstream VIF[0]. Ignoring.
Feb 23 12:43:58 ubnt udm-iptv[19295]: The IGMP message was local multicast. Ignoring.
Feb 23 12:43:58 ubnt udm-iptv[19295]: RECV V2 member report   from 192.168.1.62    to 225.0.71.1
Feb 23 12:43:58 ubnt udm-iptv[19295]: Updated route entry for 225.0.71.1 on VIF #0
Feb 23 12:43:59 ubnt udm-iptv[19295]: RECV V2 member report   from 192.168.1.144   to 239.255.255.251
Feb 23 12:43:59 ubnt udm-iptv[19295]: Updated route entry for 239.255.255.251 on VIF #0
Feb 23 12:43:59 ubnt udm-iptv[19295]: RECV V2 member report   from 192.168.1.81    to 233.89.188.1
Feb 23 12:43:59 ubnt udm-iptv[19295]: Updated route entry for 233.89.188.1 on VIF #0
Feb 23 12:43:59 ubnt udm-iptv[19295]: The source address 192.168.1.81 for group 233.89.188.1 is from downstream VIF[0]. Ignoring.

**Comment by fabianishere** [comment]
Is your STB connected to the correct VLAN (44) in the first case?

**Reply by APellie** [reply]
Yes it is

**Reply by fabianishere** [reply]
How did you do this? Could you check the IP address of the STB?

**Reply by APellie** [reply]
The STB was connected on port 2 of the UDM Pro and configured with IPTV network only. It got IP address 192.168.2.57.
It worked for couple of minutes and then crashes. after restarting (systemctl restart udm-iptv) it takes just a few seconds before IPTV freezes again. 

Weird thing is that now with everything running over default network (LAN) everything is working.
IP Address: 192.168.1.62 and IPTV IP address: 10.194.0.154

**Reply by fabianishere** [reply]
Perhaps some firewall rule causing trouble?

**Reply by APellie** [reply]
firewall rules for both networks are the same

**Reply by APellie** [reply]
Performed a reset of the whole environment. Using the 2.0.5 package on the default LAN works straight away. IPTV is stable.
Then after creating a new network (called it LAN2 this time) and following the installation process I can't get a stable IPTV.
Any logging that I can study on the UDM Pro?

**Reply by fabianishere** [reply]
Create a packet trace of the traffic from your STB? E.g., `tcpdump -i br44 host 192.168.2.57`

**Reply by RG-Bull** [reply]
Hi @fabianishere and @APellie 
My setup:
ISP: KPN
Router: UDM Pro Firmware Version 1.11.0
Switch: USW 24 Poe Firmware Version 5.76.7

I  had the same problem.
Ok I am a starter in this kinda way of configurate network etc. But I followed all steps. Is it a good way to keep it in this configuration or should I/we search for a better solution?
Could this give problems on the rest of the network?
